resources/license_banner.txt

```diff
@@ -1,4 +1,4 @@
-FingerprintJS v<%= pkg.version %> - Copyright (c) FingerprintJS, Inc, <%= new Date().getFullYear() %> (https://fingerprintjs.com)
+FingerprintJS v<%= pkg.version %> - Copyright (c) FingerprintJS, Inc, <%= new Date().getFullYear() %> (https://fingerprint.com)
```

Comment:
Shouldn't be this copyright just `Fingerprint`?

```suggestion
FingerprintJS v<%= pkg.version %> - Copyright (c) Fingerprint, Inc, <%= new Date().getFullYear() %> (https://fingerprint.com)
```

Reply:
The company name hasn’t changed. Only the domain and the brand names have changed.